packages/xo-web/src/common/render-xo-item.js

```diff
@@ -170,19 +170,29 @@ Vm.defaultProps = {
 export const VmTemplate = decorate([
   connectStore(() => {
     const getObject = createGetObject()
+    const getPool = createGetObject(
+      createSelector(
+        getObject,
+        vm => get(() => vm.$pool)
+      )
+    )
     return (state, props) => ({
       // FIXME: props.self ugly workaround to get object as a self user
       template: getObject(state, props, props.self),
+      pool: getPool(state, props),
     })
   }),
-  ({ id, template }) => {
+  ({ id, template, pool, self }) => {
     if (template === undefined) {
       return unknowItem(id, 'template')
     }
 
     return (
       <span>
         <Icon icon='vm' /> {template.name_label}
+        {!self && pool !== undefined && (
```

Comment:
You don't need the `!self` condition as `pool` will be `undefined` for self users anyway.